inc/common/purge.php

```diff
@@ -218,16 +227,18 @@ function rocket_clean_post( $post_id ) {
 	 *
 	 * @since 1.3.0
 	 *
-	 * @param obj    $post       The post object
-	 * @param array  $purge_urls URLs cache files to remove
-	 * @param string $lang       The post language
+	 * @param WP_Post $post       The post object
+	 * @param array   $purge_urls URLs cache files to remove
+	 * @param string  $lang       The post language
 	 */
 	do_action( 'after_rocket_clean_post', $post, $purge_urls, $lang );
+
+	$done[ $post_id ] = 1;
```

Comment:
This should go right after the `isset()` test imho, otherwise this is will run multiple times if we don't reach this point.